README.md

```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.com/sbt/sbt.svg?branch=develop)](https://travis-ci.com/github/sbt/sbt)
 [![Latest version](https://img.shields.io/github/tag/sbt/sbt.svg)](https://index.scala-lang.org/sbt/sbt)
+[![Scaladoc](https://javadoc.io/badge2/org.scala-sbt/sbt/scaladoc.svg)](https://javadoc.io/doc/org.scala-sbt/sbt)
```

Comment:
Why not use the official scaladoc linked from https://www.scala-sbt.org/learn.html? Unlike the one you linked the official one is a unidoc that includes all the modules.